src/routes/transactionWrapper.js

```diff
@@ -1,9 +1,22 @@
 import { sequelize } from '../models';
+import handleErrors from '../lib/apiErrorHandler';
+
+const namespace = 'SERVICE:WRAPPER';
+
+const logContext = {
+  namespace,
+};
 
 export default function transactionWrapper(originalFunction) {
   return async function wrapper(req, res, next) {
     return sequelize.transaction(async () => {
-      const result = await originalFunction(req, res, next);
+      let result;
```

Comment:
knowing that the transaction functionally handles all the errors as a try/catch, why do we want to change this file? What purpose is this serving? I'm concerned that adding that "Wrapper" namespace would make it harder to track down an error in the future

Reply:
So your suggestion is to remove line 11 and line 22, correct?

Reply:
Yes to get the error into the handleErrors function

Reply:
> knowing that the transaction functionally handles all the errors as a try/catch, why do we want to change this file? What purpose is this serving? I'm concerned that adding that "Wrapper" namespace would make it harder to track down an error in the future

We have a mixture of cases in the code where we already handle any errors and the ones that don't. Where we handle them correctly, the namespace will be correct (non-wrapper). It'd be best to refactor the individual functions that are missing try/catch, but that's a bigger change.